docs/userguide.md

```diff
@@ -247,6 +247,72 @@ done
 If you were able to verify the resources removed and added back then you have
 successfully verified a working federation-v2 deployment.
 
+In addition to specifying an explicit list of clusters that a resource should be
+propagated to via the `spec.clusterNames` field of a placement resource, it is possible
+to use the `spec.clusterSelector` field to provide a label selector that determines
+that list of clusters dynamically.
```

Comment:
Ah, I think I see the reason for @tghartland's comment below.  There needs to be terminology to clearly differentiate the results of 'scheduling' from the results of 'label selection'.  I think the following could work:

`s/dynamically/at runtime/`

Reply:
Yes, this is where I was getting the idea that dynamic = label selection. Using "at runtime" here would clear up some of the confusion.

Reply:
Done